charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -185,7 +182,7 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
-          {{- with default .Values.resources (default .Values.controller.resources .Values.controller.containerResources.attacher) }}
+          {{- with default .Values.controller.resources .Values.controller.containerResources.attacher }}
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
```

Comment:
IMO we should remove the wrapper around csi-snapshotter and  remove `enableVolumeSnapshot`

As long as we document that having the CRDs and snapshot-controller is a prerequisite, and IMO it makes sense to include the same note in README as in NOTES.txt, users should not be surprised to see csi-snapshotter is failing.

(Sorry to sound like a broken record on this issue, really want to be rid of it!)